games-engines/scummvm/scummvm-1.9.0.recipe

```diff
@@ -56,41 +58,56 @@ BUILD_REQUIRES="
 	devel:libogg$secondaryArchSuffix
 	devel:libpng$secondaryArchSuffix
 	devel:libSDL$secondaryArchSuffix
+	#devel:libsdl_net$secondaryArchSuffix #not needed yet
 	devel:libtheora$secondaryArchSuffix
 	devel:libvorbis$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
-"
+	"
 
 BUILD_PREREQUIRES="
 	cmd:gcc$secondaryArchSuffix
-	cmd:git
 	cmd:ld$secondaryArchSuffix
-	cmd:libtoolize
+	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
 	cmd:nasm
 	cmd:pkg_config$secondaryArchSuffix
-"
+	"
 
 BUILD()
 {
 	libtoolize --force --copy --install
 	CPPFLAGS=`freetype-config --cflags` ./configure --prefix=$prefix \
-	--bindir=$appsDir --libdir=$libDir --datarootdir=$dataDir \
-	--mandir=$manDir --docdir=$docDir \
-	--disable-debug --enable-release #--disable-all-engines
-	#--disable-all-engines can be used to only compile scummvm without
-	#engines, which makes it build faster, those that are still under
-	#construction can be built with --enable-all-engines for testing purposes
-
+		--bindir=$appsDir --libdir=$libDir --datarootdir=$dataDir \
+		--mandir=$manDir --docdir=$docDir \
+		--disable-debug --enable-release --disable-eventrecorder \
+		#--disable-all-engines #can be used to compile scummvm without \
+		#engines, which makes it build faster (to test), those that \
+		#are still under construction can be built with --enable-all-engines \
+		#for testing purposes
 	make $jobArgs
 }
 
 INSTALL()
 {
 	make install
 	mv $appsDir/scummvm $appsDir/ScummVM
-	addResourcesToBinaries $portDir/additional-files/scummvm.rdef \
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		$portDir/additional-files/scummvm.rdef > scummvm.rdef
```

Comment:
Sorry for this late reply. I was previously running a system which lacks a firewall :-(

> Thanks for the reply, will try to change it today,

Cool!

> ps, do I also need to "git add" the recipe before committing?

No, you don't need to "git add" because you have already run:
```
git mv scummvm_tools-1.8.0.recipe scummvm_tools-1.9.0.recipe
```
which has the same effect of:
```
cp scummvm_tools-1.8.0.recipe foobar
git rm scummvm_tools-1.8.0.recipe
mv foobar scummvm_tools-1.9.0.recipe
git add scummvm_tools-1.9.0.recipe
```
Of course, in all cases one still needs to:
```
git commiy --amend scummvm_tools-1.8.0.recipe scummvm_tools-1.9.0.recipe
```
(which you did, correctly.)